README.rst

```diff
@@ -266,11 +266,11 @@ to the request:
 
     @responses.activate
     def my_func():
-        req_data = {"some": "other", "data": "fields"}
         req_files = {"file_name": b"Old World!"}
+        req_data = {"some": "other", "data": "fields"}
```

Comment:
what is the sense to move this line?

Reply:
Ah, nothing much. Only maintaining the order they appear in args. Can undo this if you want.

Reply:
less changes is always preferable, it allows to check PR easier and not to spend time understanding why change was done. Please undo this line

Reply:
Done.